toqito/helper/bell_notation_conversions.py

```diff
@@ -0,0 +1,748 @@
+"""Conversions between Bell inequality notations."""
+
+import numpy as np
+
+
+def cg_to_fc(cg_mat: np.ndarray, behaviour: bool = False) -> np.ndarray:
+    r"""Convert a Bell functional or behaviour from Collins-Gisin (CG) to Full Correlator (FC) notation.
+
+    The Collins-Gisin (CG) notation for a Bell functional or behaviour is represented by a matrix:
+
+    .. math::
+        \text{CG} =
+        \begin{pmatrix}
+            K      & p_B(0|1) & p_B(0|2) & \dots \\
+            p_A(0|1) & p(00|11) & p(00|12) & \dots \\
+            p_A(0|2) & p(00|21) & p(00|22) & \dots \\
+            \vdots   & \vdots   & \vdots   & \ddots
+        \end{pmatrix}
+
+    The Full Correlator (FC) notation is represented by:
+
+    .. math::
+        \text{FC} =
+        \begin{pmatrix}
+            K      & \langle B_1 \rangle & \langle B_2 \rangle & \dots \\
+            \langle A_1 \rangle & \langle A_1 B_1 \rangle & \langle A_1 B_2 \rangle & \dots \\
+            \langle A_2 \rangle & \langle A_2 B_1 \rangle & \langle A_2 B_2 \rangle & \dots \\
+            \vdots   & \vdots      & \vdots      & \ddots
+        \end{pmatrix}
+
+    This function converts between these two notations.
+
+    Examples
+    ==========
+
+    Consider the CHSH inequality in CG notation for a functional:
+
+    .. math::
+        \text{CHSH}_{CG} =
+        \begin{pmatrix}
+            0 & 0 & 0 \\
+            0 & 1 & -1 \\
+            0 & -1 & 1
+        \end{pmatrix}
+
+    Converting to FC notation:
+
+    >>> import numpy as np
+    >>> from toqito.helper import cg_to_fc
+    >>> chsh_cg = np.array([[0, 0, 0], [0, 1, -1], [0, -1, 1]])
+    >>> cg_to_fc(chsh_cg)
+    array([[ 0.  ,  0.  ,  0.  ],
+           [ 0.  ,  0.25, -0.25],
+           [ 0.  , -0.25,  0.25]])
+
```

Comment:
we are switching to using executable examples in docstrings. Take a look at the WIP in #1180 

You might have to rebase your branch if you started working on this PR a while ago. Executable code blocks for tutorials were added in another PR.